src/Hosted/openPosition.html

```diff
@@ -0,0 +1,165 @@
+﻿<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Open Position</title>
+    <meta name="description" content="" />
+    <script src="themekit/scripts/jquery.min.js"></script>
+    <script src="themekit/scripts/main.js"></script>
+    <link rel="stylesheet" href="themekit/css/bootstrap-grid.min.css" />
+    <link rel="stylesheet" href="themekit/css/style.min.css" />
+    <link rel="stylesheet" href="themekit/css/magnific-popup.min.css" />
+    <link rel="stylesheet" href="themekit/css/contact-form.min.css" />
+    <link rel="stylesheet" href="themekit/css/content-box.min.css" />
+    <link
+      rel="stylesheet"
+      href="themekit/media/icons/iconsmind/line-icons.min.css"
+    />
+    <link rel="stylesheet" href="assets/skin.min.css" />
+    <link rel="stylesheet" href="assets/custom.min.css" />
+    <link rel="shortcut icon" href="assets/favicon.ico" type="image/x-icon" />
+  </head>
+  <body>
+      <div id="preloader"></div>
+      <nav class="menu-classic menu-transparent light menu-fixed menu-one-page
+    align-right menu-wide" data-menu-anima="fade-bottom"
+           data-scroll-detect="true">
+          <div ws-replace="MenuBar"></div>
+      </nav>
+      <main>
+          <section class="section-base">
+              <div class="container">
+                  <div class="row">
+                      <div class="col-lg-6">
+                          <hr class="space-xs" />
+                          <h1 style="margin-top: 0">F# Developer</h1>
+                          <p>
+                              What you get:
+                          </p>
+                          <ul>
+                              <li>Variety and Challenge</li>
+                              <li>Work-Life-Balance</li>
+                              <li>Professional and Personal Growth</li>
+                          </ul>
+                      </div>
+                      <div class="col-lg-6">
+                          <img class="align-center"
+                               src="http://via.placeholder.com/1019x608" />
+                      </div>
+                  </div>
+                  <hr class="space-lg" />
+                  <div class="row">
+                    <div class="col-lg-4">
+                        <div class="icon-box icon-box-left">
+                            <i class="im-air-balloon"></i>
```

Comment:
@erolaliyev We should not use the iconsmind icon set, we should use icons from the Font Awesome set to achieve consistency. (We need Font Awesome for the social icons). At the moment if I'm not mistaken the following pages still have iconsmind icons:

- openPosition.html
- careers.html
- trainings.html